src/cf-forms/src/molecules/input-with-btn.less

```diff
@@ -0,0 +1,49 @@
+//
+// Input with button
+//
+
+.m-input-with-btn {
+
+    .respond-to-min( 480px, {
+        .grid_nested-col-group();
+    });
+
+    &_input {
```

Comment:
Can we make this `&_input-col` or `&_input-container` to clarify it's not the input itself, but a wrapper of the input? Same for the button below.